yt/visualization/plot_modifications.py

```diff
@@ -1960,11 +1962,11 @@ def __call__(self, plot):
         px, py = self._convert_to_plot(plot, [px, py])
 
         # Convert halo radii to a radius in pixels
-        radius = halo_data[self.radius_field][:].in_units(units)
+        radius = halo_data[("all", self.radius_field)][:].in_units(units)
```

Comment:
I'm not sure this is right.  @brittonsmith what do you think?  Is there a specific halo type that we should use for the `HaloCatalog` particle types?